README.md

```diff
@@ -1,9 +1,32 @@
-## Glitch Grid
+## Motivation
 
-This is a toy distributed system which we will use to demonstrate how to
-transform your system to run within [Antithesis](https://antithesis.com/). We have the same
-basic system implemented in Go and Python, as well as a language-agnostic
-workload which can be used to test the system.
+Glitch Grid is a toy distributed system for demonstrating how to
+interface your system with the [Antithesis Platform](https://antithesis.com/). This project demonstrates:
+
+* Use of an SDK for [assertion definition](https://antithesis.com/docs/using_antithesis/properties.html)
+   and control flow.
+* Build and, for Go, [source instrumentation](https://antithesis.com/docs/instrumentation/go_instrumentation.html)
+* Triggering tests and displaying results via [CI Integrations](https://antithesis.com/docs/using_antithesis/ci.html)
+
+Why demonstrate on a toy project? In order to have interesting test results, we wanted
+to have a software system that contains bugs! Welcome, GlitchGrid.
+
+Please refer to the [Antithesis Docs](https://antithesis.com/docs/) for more information about
+how to get started with Antithesis, best practrices, etc.
+
+### SDK Use
+
+This project includes [a variety of assertions](https://github.com/search?q=repo%3Aantithesishq%2Fglitch-grid+%28Always+OR+Sometimes%29&type=code)
```

Comment:
First sentence is a run-on sentence, and also introduces an entire concept (test properties) that is very complicated and not directly relevant to this repo! 

I would rephrase with something like: "This project demonstrates how to use the Antithesis SDK to add assertions about your software. This includes conventional assertions, and also Sometimes Assertions [LINK] which can help you assess the quality of your testing or check for unreachable code."